Makefile

```
@@ -43,6 +53,25 @@ clean-db:  ## remove LMS databases
 	$(COMPOSE) rm edx_mongodb edx_mysql edx_redis 
 .PHONY: clean-db
 
+install-tests: ## install tests dependencies
+	@$(YARN) install
```

Maybe we should be explicit on this and not hide executed command, WDYT?

```suggestion
	$(YARN) install
```